src/api/app/views/layouts/webui2/_login_form.html.haml

```diff
@@ -12,12 +12,11 @@
           = hidden_field_tag(:proxypath, 'reserve')
           = hidden_field_tag(:message, 'Please log in')
           = hidden_field_tag(:url, request.fullpath)
-        .form-group
-          = label_tag(:username, 'Username', class: 'text-light')
-          = text_field_tag(:username, nil, placeholder: 'User Name', required: true, class: 'form-control')
-        .form-group
-          = label_tag(:password, 'Password', class: 'text-light')
-          = password_field_tag(:password, nil, placeholder: 'Password', required: true, class: 'form-control')
-        .float-right
-          = submit_tag('Log In', class: 'btn btn-success')
         .clearfix
+          .form-group
+            = label_tag(:username, 'Username', class: 'text-light')
+            = text_field_tag(:username, nil, placeholder: 'User Name', required: true, class: 'form-control')
+          .form-group
+            = label_tag(:password, 'Password', class: 'text-light')
+            = password_field_tag(:password, nil, placeholder: 'Password', required: true, class: 'form-control')
+          = submit_tag('Log In', class: 'btn btn-success float-right')
```

Comment:
`.clearfix` is only for the submit button, so I believe it should be above this line. The other form controls are not floating.

Reply:
Right! Done. Please review again.